drivers/gpio/gpio_cc2650.c

```diff
@@ -174,7 +174,7 @@ static int gpio_cc2650_config_pin(int pin, int flags)
 	if (flags & GPIO_DS_DISCONNECT_LOW) {
 		disconnect(pin, &gpio_doe31_0_config, &iocfg_config);
 	}
-	if (flags & GPIO_DS_DFLT_LOW) {
+	if (!(flags & GPIO_DS_ALT_LOW)) {
```

Comment:
How about we code this as:

if ((GPIO_DS_DFLT_LOW == (flags & GPIO_DS_ALT_LOW)) {